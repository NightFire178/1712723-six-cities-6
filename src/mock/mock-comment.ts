import mockUser from "./mock-user";
import comment from '../types/comment'

const mockComment:comment = {
  comment: `Beautiful space, fantastic location and atmosphere,
  really a wonderful place to spend a few days. Will be back.`,
  date: `2021-03-07T08:04:28.647Z`,
  id: 10,
  rating: 4,
  user: mockUser
}
export default mockComment

export const mockArrayComments:Array<comment> =[
  {
    comment: `Beautiful space, fantastic location and atmosphere,
  really a wonderful place to spend a few days. Will be back.`,
    date: `2021-03-07T08:04:28.647Z`,
    id: 15,
    rating: 3,
    user: mockUser
  },
  {
    comment: `Beautiful space, fantastic location and atmosphere,
  really a wonderful place to spend a few days. Will be back.`,
    date: `2021-03-07T08:04:28.647Z`,
    id: 13,
    rating: 5,
    user: mockUser
  },
  {
    comment: `What an amazing view! The house is stunning and in
    an amazing location. The large glass wall had an amazing view of the river!`,
    date: `2021-03-07T08:04:28.647Z`,
    id: 10,
    rating: 4,
    user: mockUser
  }
]
