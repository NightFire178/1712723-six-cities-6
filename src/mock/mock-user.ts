import User from "../types/user";

const mockUser:User = {
  id: 1,
  name: `vasa`,
  is_pro: true,
  avatar_url:`https://assets.htmlacademy.ruâ€¦tar/6.jp`,
  email: `sdfhskd@maid.ru`
}
export default mockUser
