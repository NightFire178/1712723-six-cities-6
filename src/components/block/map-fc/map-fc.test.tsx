import  React from 'react'
import {render} from '@testing-library/react'
import MapFc from "./map-fc";
import {mockArrayHotels} from "../../../mock/mock-hotels";


test(`mapFC component block test`, ()=>{
  const hotels = JSON.parse(JSON.stringify(mockArrayHotels))
  const {container} = render(<MapFc hotels={hotels} />)
  expect(container).toMatchSnapshot()
})
