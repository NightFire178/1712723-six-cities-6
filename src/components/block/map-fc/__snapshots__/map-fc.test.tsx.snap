// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mapFC component block test 1`] = `
<div>
  <div
    class="leaflet-container leaflet-grab leaflet-touch-drag"
    id="map"
    style="height: 100%; position: relative;"
    tabindex="0"
  >
    <div
      class="leaflet-pane leaflet-map-pane"
      style="left: 0px; top: 0px;"
    >
      <div
        class="leaflet-pane leaflet-tile-pane"
      >
        <div
          class="leaflet-layer "
          style="z-index: 1;"
        >
          <div
            class="leaflet-tile-container leaflet-zoom-animated"
            style="z-index: 18; left: 0px; top: 0px;"
          >
            <img
              alt=""
              class="leaflet-tile"
              role="presentation"
              src="https://b.basemaps.cartocdn.com/rastertiles/voyager/13/4195/2748.png"
              style="width: 256px; height: 256px; left: -6px; top: -9px;"
            />
          </div>
        </div>
      </div>
      <div
        class="leaflet-pane leaflet-shadow-pane"
      />
      <div
        class="leaflet-pane leaflet-overlay-pane"
      />
      <div
        class="leaflet-pane leaflet-marker-pane"
      >
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: -29px; top: 19px; z-index: 19;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 15206px; top: -958px; z-index: -958;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 134px; top: -27px; z-index: -27;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 3136px; top: -14230px; z-index: -14230;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: -87px; top: 176px; z-index: 176;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 14055px; top: -3407px; z-index: -3407;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 15142px; top: -1023px; z-index: -1023;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 32780px; top: -25854px; z-index: -25854;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 33036px; top: -25668px; z-index: -25668;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 3264px; top: -14230px; z-index: -14230;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: -11716px; top: 17938px; z-index: 17938;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 32762px; top: -25502px; z-index: -25502;"
          tabindex="0"
        />
        <img
          alt=""
          class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
          src="img/pin.svg"
          style="margin-left: -13.5px; margin-top: -19.5px; width: 27px; height: 39px; left: 32902px; top: -25786px; z-index: -25786;"
          tabindex="0"
        />
      </div>
      <div
        class="leaflet-pane leaflet-tooltip-pane"
      />
      <div
        class="leaflet-pane leaflet-popup-pane"
      />
    </div>
    <div
      class="leaflet-control-container"
    >
      <div
        class="leaflet-top leaflet-left"
      />
      <div
        class="leaflet-top leaflet-right"
      />
      <div
        class="leaflet-bottom leaflet-left"
      />
      <div
        class="leaflet-bottom leaflet-right"
      >
        <div
          class="leaflet-control-attribution leaflet-control"
        >
          <a
            href="https://leafletjs.com"
            title="A JS library for interactive maps"
          >
            Leaflet
          </a>
           | © 
          <a
            href="https://www.openstreetmap.org/copyright"
          >
            OpenStreetMap
          </a>
           contributors © 
          <a
            href="https://carto.com/attributions"
          >
            CARTO
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
